Dockerfile

```diff
@@ -5,19 +5,13 @@ ARG BASE_IMAGE=python:3.10-slim
 
 FROM ${BASE_IMAGE} AS base
 
-RUN apt-get update
-RUN apt-get install -y apt-transport-https ca-certificates gnupg curl gcc g++
+RUN apt-get update && apt-get install -y curl gnupg
```

Comment:
Can you explain this change? Is it about running `apt-get update &&` before *every* `apt-get install`? Why isn't it enough to run `apt-get update` once in the beginning?

Reply:
Sometimes the first layer gets cached and the that causes stale package lists. This is a standard best practice when writing Dockerfiles. Here is more info about the issue and why: https://docs.docker.com/build/building/best-practices/#apt-get

Reply:
Got it. Thanks! Could you add this comment and link as comments?

Reply:
done, added another comment.

Reply:
I don't see the comment. Maybe github doesn't have the latest commit?

Reply:
Sorry about that.. I wish I could blame GitHub but looks like I simply was being silly. I didn't push...

The commit should show up now.